src/NzbDrone.Core/Parser/QualityParser.cs

```diff
@@ -40,17 +40,49 @@ public class QualityParser
                                                                 RegexOptions.Compiled);
 
         private static readonly Regex BitRateRegex = new Regex(@"(?:
-                                                                  (?<B192>192[ ]?kbps)|(?<B192>192$)|(?<B192>[\[\(].*192.*[\]\)])|
+                                                                  (?<B192>192[ ]?kbps)|(?<B192>192$)|(?<B192>[\[\(].*192.*[\]\)])|(?<MP3>MPEG Version \d+ Audio, Layer 3$)|
                                                                   (?<B256>256[ ]?kbps)|(?<B256>256$)|(?<B256>[\[\(].*256.*[\]\)])|
                                                                   (?<B320>320[ ]?kbps)|(?<B320>320$)|(?<B320>[\[\(].*320.*[\]\)])|
                                                                   (?<B512>512[ ]?kbps)|(?<B512>512$)|(?<B512>[\[\(].*512.*[\]\)])|
                                                                   (?<Flac>flac[-_.\]\b)} ])|(?<Flac>flac$)|
-                                                                  (?<VBR>VBR[ ]?kbps)|(?<VBR>VBR$)|(?<VBR>[\[\(].*VBR.*[\]\)])
+                                                                  (?<VBR>VBR[ ]?kbps)|(?<VBR>VBR$)|(?<VBR>[\[\(].*VBR.*[\]\)])|(?<VBR>MPEG Version \d+ Audio, Layer 3 VBR$)
                                                                   )",
                                                                   RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
 
-        private static readonly Regex CodecRegex = new Regex(@"\b(?:(?<x264>x264)|(?<h264>h264)|(?<xvidhd>XvidHD)|(?<xvid>Xvid)|(?<divx>divx))\b",
-                                                                RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex CodecRegex = new Regex(@"(?:
+                                                                  (?<MP3>MPEG Version \d+ Audio, Layer 3$)|
+                                                                  (?<Flac>flac[-_.\]\b)} ])|(?<Flac>Flac$)|
+                                                                  (?<VBR>[\[\(].*VBR.*[\]\)])|(?<VBR>MPEG Version \d Audio, Layer 3 VBR$)
+                                                                  )",
+            RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
+
+
+        public static QualityModel ParseQuality(string desc, int bitrate, int sampleRate)
+        {
+            var result = new QualityModel { Quality = Quality.Unknown };
+
+            switch (bitrate)
+            {
+                case 192:
+                    result.Quality = Quality.MP3192;
```

Comment:
Might want to rename the Enum to something like MP3_192 to avoid confusing to what part the 3 belongs.